SECURITY.md

```diff
@@ -0,0 +1,24 @@
+# Security Policy
+
+## Supported Versions
+<!--
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.1.x   | :white_check_mark: |
+| 5.0.x   | :x:                |
+| 4.0.x   | :white_check_mark: |
+| < 4.0   | :x:                |
+-->
```

Comment:
What does this imply exactly ?

Reply:
I commented it out: we could use it just in case we would like to support vulnerabilities in specific versions. As we are not backporting security fixes to previous minor versions, maybe we should explicitly mention that our support only covers the major version.

Reply:
Yes supporting vulnerability updates for each major version sounds reasonable.
```suggestion
-->

Each major version is currently being supported with security updates.
```